opentelemetry-sdk/src/opentelemetry/sdk/_logs/_internal/__init__.py

```diff
@@ -491,7 +491,15 @@ def _get_attributes(record: logging.LogRecord) -> Attributes:
             if exctype is not None:
                 attributes[SpanAttributes.EXCEPTION_TYPE] = exctype.__name__
             if value is not None and value.args:
-                attributes[SpanAttributes.EXCEPTION_MESSAGE] = value.args[0]
+                if type(value.args[0]) is str:
+                    attributes[SpanAttributes.EXCEPTION_MESSAGE] = value.args[
+                        0
+                    ]
+                else:
```

Comment:
Is it safe to assume that the only other type is an `Exception` and that it is contained within `value.args[0].args` for ALL cases and not just `requests`?

Reply:
👆 this is a good point. As written, the code doesn't handle yet more nesting.

Would it work to say `attributes[SpanAttributes.EXCEPTION_MESSAGE] = str(value)` as Leighton mentioned? This appears to produce what we want (the exception message) regardless of nesting.

Reply:
Valid point, I agree with you that we should handle another nested exception cases. 
Now the codes handle them by converting the first argument to string as you suggested, and whether it will work or not in all cases can be checked in my test.

Reply:
I simply tested your idea:
```
>>> str(ZeroDivisionError())
''
>>> str(ZeroDivisionError(ZeroDivisionError()))
''
>>> str(ZeroDivisionError(ZeroDivisionError("asd")))
'asd'
```
It seems that the stringfy also works for the nested exception cases.